packages/core/src/vendor/zippa/zipper.js

```diff
@@ -60,7 +60,7 @@ const TOPPATH = {
  * @class Zipper
  * @namespace Zipper
  */
-function Zipper(item, path, meta) {
+export function Zipper(item, path, meta) {
```

Comment:
We don't want this change when all methods are implemented in skele, regarding zipper.js.